packages/twenty-front/src/modules/views/hooks/useViewBarFilterHotKeys.ts

```diff
@@ -0,0 +1,23 @@
+import { useDropdown } from '@/ui/layout/dropdown/hooks/useDropdown';
+import { useGlobalHotkeys } from '@/ui/utilities/hotkey/hooks/useGlobalHotkeys';
+import { AppHotkeyScope } from '@/ui/utilities/hotkey/types/AppHotkeyScope';
+import { VIEW_BAR_FILTER_DROPDOWN_ID } from '@/views/constants/ViewBarFilterDropdownId';
+import { useSearchFilter } from '@/views/hooks/useSearchFilter';
+
+export const useViewBarFilterHotKeys = () => {
+  const { openDropdown } = useDropdown(VIEW_BAR_FILTER_DROPDOWN_ID);
+  const { setShowSearchInput, setSearchInputValueFromExistingFilter } =
+    useSearchFilter(VIEW_BAR_FILTER_DROPDOWN_ID);
+
+  useGlobalHotkeys(
```

Comment:
Let's not use a global hotkey here, it is not, we don't have a view bar in settings.

When you encounter this problem, you should ask yourself what underlying module should listen to this, maybe record index in this case ?